keps/1136-provisioning-request-support/README.md

```diff
@@ -193,6 +193,33 @@ type ProvisioningRequestConfigSpec struct {
 	// +optional
 	// +kubebuilder:default={backoffLimitCount:3,backoffBaseSeconds:60,backoffMaxSeconds:1800}
 	RetryStrategy *ProvisioningRequestRetryStrategy `json:"retryStrategy,omitempty"`
+
+	// podSetUpdates specifies the update of the workload's PodSetUpdates which
```

Comment:
I'm a little confused by the comment - podSetUpdates is specified twice

Reply:
Because this new field is used to set the pre-existing field in workload: https://github.com/kubernetes-sigs/kueue/blob/9c385afb2f2def72d3a2147826d8580067f850cc/apis/kueue/v1beta1/workload_types.go#L425